index.html

```diff
@@ -5,9 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Awesome Books</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     
+    <section id='booksSection'>
+        <h2>Books</h2>
+    </section>
+
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code cleaner.